usage-examples/test-execution.md

```diff
@@ -0,0 +1,44 @@
+# Test Execution Example
+This example presents how Eiffel events sent as part of a test activity can be connected.
+
+## Introduction
+Using Eiffel events to describe the execution of a test activity can be done in wide variety of ways. On one edge, a test activity can be described with Eiffel activity events only, and on the other edge a test activity can be expanded to send a big set of Eiffel events revealing the inner details of the test activity. Which way to choose depends on the complexity of your test execution system and on the need to describe and maybe visualize the nitty-gritty details of the test activity.
+
+The Eiffel protocol provides several events to use within a test activity. This example shows how to make use of most of the Eiffel events related to test activities, with the purpose to present how they all are related.
```

Comment:
I think we need to explain the context a bit for TERC to make sense to people. I've been thinking of doing this in a screen cast, but haven't had time yet. This would also be a good place.

E.g. "To understand the example provided here, it is important to explain a few concepts. While Eiffel does not make any assumptions about the underlying infrastructure and/or testing methodology, it does encourage separation of concerns. In this view, it is not the responsibility of the activity orchestrator (e.g. the CI server) to determine the test scope, nor is it the responsibility of the test executor. Instead, the EiffelTestExecutionRecipeCollectionEvent makes it possible for a third actor to determine the test scope, which is then handed off to a test executor."